lib/aruba/processes/spawn_process.rb

```diff
@@ -83,7 +83,7 @@ def start
             sleep startup_wait_time
           end
         rescue ChildProcess::LaunchError => e
-          raise LaunchError, "It tried to start #{cmd}. " + e.message
+          raise LaunchError, "It tried to start #{commandline}. " + e.message
```

Comment:
(`cmd` is passed to the init method - and not to this one. Probably the init was split into two methods some time ago.)